.template/variants/web/app/javascript/build.js

```diff
@@ -1,13 +1,17 @@
 const watch = process.argv.slice(2).includes('--watch');
 
-require('esbuild').build({
-  entryPoints: ['app/javascript/application.js'],
-  inject: ['app/javascript/global.js'],
-  bundle: true,
-  sourcemap: true,
-  watch: watch,
-  outdir: 'app/assets/builds',
-}).catch((error) => {
-  console.error(error);
-  process.exit(1);
-});
+require('esbuild')
+  .context({
+    entryPoints: ['app/javascript/application.js'],
+    inject: ['app/javascript/global.js'],
+    bundle: true,
+    sourcemap: true,
+    outdir: 'app/assets/builds',
+  })
+  .then((ctx) => {
+    watch ? ctx.watch() : ctx.rebuild().then(() => process.exit(0));
+  })
+  .catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
```

Comment:
```suggestion
context = {
    entryPoints: ['app/javascript/application.js'],
    inject: ['app/javascript/global.js'],
    bundle: true,
    sourcemap: true,
    outdir: 'app/assets/builds',
}
  
require('esbuild')
  .context(context)
  .then((ctx) => {
    watch ? ctx.watch() : ctx.rebuild().then(() => process.exit(0));
  })
  .catch((error) => {
    console.error(error);
    process.exit(1);
  });
```

Reply:
It would help to know which benefit you want to bring with this change 😅 It looks okay, but not sure if it brings value.

There are open source tools ([example](https://github.com/bridgetownrb/bridgetown/blob/dd004e0c788d62c0381189ade475c6b18489aa73/bridgetown-core/lib/bridgetown-core/commands/esbuild/esbuild.defaults.js.erb#L191-L215)) that have far more complex Esbuild config, but the config hash is not extracted.

Reply:
We are too much into Ruby and haunted by Rubocop and Reek these days xD